src/css/_global.scss

```diff
@@ -1,32 +1,32 @@
-html, body {
+html,
+body {
   height: 100%;
   margin: 0;
   padding: 0;
   overflow: hidden;
 }
 
 html {
-  font-family: roboto, sans-serif;
-  -webkit-tap-highlight-color: rgba(0,0,0,0);
+  font-family: Roboto, sans-serif;
```

Comment:
I changed the font case because I couldn't find it at first with search. Otherwise this has no effect.